### PR TITLE
fix sat5_iso_install task

### PR DESCRIPTION
**Pull request by lpramuk**
fixes Issue #174 TypeError right after running satellite5_iso_install task

### CODE REVIEW

**Comment by elyezer** [comment]
ACK, this will fix the issue, but seems that we will need further refactor to make the product_install task install Satellite 5. Or maybe will be better to split to a new task.

File: automation_tools/__init__.py [hunk 1]
```diff
@@ -888,6 +888,9 @@ def satellite5_iso_install(
             '--re-register --run-updater=yes --enable-tftp=yes')
         run('yum -y update')
 
+    installer_options = {}
+    return installer_options
```
Comment (Ichimonji10) [inline automation_tools/__init__.py]: How about this:

```python
return {}
```

**Comment by Ichimonji10** [comment]
#174 does not clearly describe what the source of the traceback is. Without more information about the source of the problem, my gut feeling tells me that this simply papers over a deeper problem. @lpramuk, can you spend some time digging into this issue and detail your findings? If nothing else, I'd appreciate knowing what's going on.

**Comment by lpramuk** [comment]
Like elyezer said before we will need further refactor to make the product_install.
Currently we used iso_install task as a teplate for sat5_iso_install task, so it has to mimics what it returns to product_install task.
In the future we may consider to create sole sat5_product_install not to obey sat6 existing constructs....

Currently sat5_iso_install produces traceback, so let's consider this as a quick fix before further refactoring...

**Comment by Ichimonji10** [comment]
> Currently sat5_iso_install produces traceback, so let's consider this as a quick fix before any further refactoring...

I'm wary of papering over problems with quick fixes. I understand that this change would fix the traceback, and have asked for more information with that in mind.

> Currently we used iso_install task as a teplate for sat5_iso_install task, so it has to mimics what it returns to product_install task.

Aha. Now I understand.

ACK pending comment.

**Comment by omaciel** [comment]
@lpramuk I agree with what @Ichimonji10  said about 'quick fixes', so can you file a new Github issue with information about what the real fix should be and assign it to yourself?

**Comment by lpramuk** [comment]
Guys I'm on your side too. I didn't cause the need for the quick fix.
I delivered what was omitted by someone else...

Lets agree on creating new issue for code refactoring of product_install to move out of this...

**Comment by omaciel** [comment]
@lpramuk we **are** on the same side and my comment was the agreement that we want a separate issue to track this :)

**Comment by lpramuk** [comment]
we want it we get it - new Issue #177

**Comment by elyezer** [comment]
@lpramuk ping me once you incorporate the @Ichimonji10's comment se we can merge this.

**Comment by elyezer** [comment]
APT

**Comment by Ichimonji10** [comment]
ACK

**Pull request merged**